tree/treeplayer/inc/ROOT/TDFUtils.hxx

```diff
@@ -35,6 +35,16 @@ namespace ROOT {
 namespace Experimental {
 template <int D, typename P, template <int, typename, template <typename> class> class... S>
 class THist;
+
+namespace TDF {
+struct SnapshotOptions {
+  Int_t autosave   = 0;
+  Int_t compress   = 1;
+  Int_t splitlevel = 99;
+  std::string mode = "RECREATE";
```

Comment:
Data member **must** be camel case prefixed by the letter 'f'

Reply:
unfortunately true even for data members of POD structs..

Reply:
This convention even for structs is really annoying... I've changed it anyway.

Reply:
> unfortunately true even for data members of POD structs..
> This convention even for structs is really annoying... I've changed it anyway.

I have no clue why being consistent and clear would be unfortunate or annoying ... (When seing somename.XYZ you can not know without a convention or looking at the declaration whether XYZ is a member function (pointer), a data member, a static data member, a enum constant or even a nest class .... using the naming convention disambiguate this and make reading the code easier and faster ... things you both always advocate for :) )

Reply:
For simple structs such as this the `fCamelCase` is a bit noisy.
`obj.fMember` feels wrong under my fingers because 99.99% of the times you should go through getters or setters, not acces the members directly.
[google agrees](https://google.github.io/styleguide/cppguide.html#Variable_Names) that for structs there is no need to distinguish data members from normal variables.

People at Taligent obviously did not agree with this view. No big deal :smile:

Reply:
> ... a bit noisy ... for structs there is no need to distinguish ...

Fair enough, I see your point(s).  I still prefer to stay consistent (and if the struct ever becomes a class then there is less to change :) ).